### PR TITLE
AL: Improve testing for empty bill index pages

### PR DESCRIPTION
**Pull request by mileswwatkins**
This should fix some of the intermittent errors we've seen with Alabama.

### CODE REVIEW

**Comment by mileswwatkins** [comment]
Refactoring further, per suggestion of @paultag. Do not review until more commits have been pushed.

**Comment by mileswwatkins** [comment]
@paultag/@rshorey: ready to go

File: openstates/al/bills.py [hunk 1]
```diff
@@ -50,30 +49,25 @@ class ALBillScraper(BillScraper):
     DATE_FORMAT = '%m/%d/%Y'
 
     def accept_response(self, response, **kwargs):
```
Comment (rshorey) [inline openstates/al/bills.py]: add a comment about how this is a scrapelib function we're redefining for easier future maintainability?

**Comment by rshorey** [comment]
This looks OK to me. Also I learned a lot about the Alabama leg dress code by reviewing these bills :)

**Pull request merged**